src/script/broadcast/BroadcastRepository.js

```diff
@@ -32,47 +32,62 @@ z.broadcast.BroadcastRepository = class BroadcastRepository {
    * @param {ClientRepository} clientRepository - Repository for client interactions
    * @param {ConversationRepository} conversationRepository - Repository for conversation interactions
    * @param {CryptographyRepository} cryptographyRepository - Repository for all cryptography interactions
-   * @param {UserRepository} userRepository - Repository for all user interactions
```

Comment:
This is the main goal of this PR: removing the dependency to the UserRepository from the BroadcastRepository.